packages/opencensus-core/package.json

```diff
@@ -48,14 +47,13 @@
     "gts": "^0.5.4",
     "mocha": "^5.0.4",
     "ncp": "^2.0.0",
+    "nyc": "11.6.0",
     "ts-node": "^4.0.0",
-    "typescript": "~2.6.1",
-    "nyc": "11.6.0"
+    "intercept-stdout": "^0.1.2",
```

Comment:
Can you preserve the alphabetical ordering here?

Reply:
Sure.